README.md

```diff
@@ -97,6 +97,14 @@ Options:
                            First extension will always be preferred to homonyms
                            with another allowed extension.
                                                        [array] [default: ["js"]]
+  --implicitDefault, -m    Uses defaults as implicit instead of named.
+                           export { default as thing } from './thing.js'
```

Comment:
Use metasyntactic variable for placeholder files/ variables, i.e. `./foo.js`.

Reply:
Ah, that sounds good. Will foo / bar it up.